sequential/backend/freq_miner.cpp

```diff
@@ -32,7 +32,8 @@ double mm=0;
 vector<vector<int>> Freq_miner(vector<Pattern*>* dfs_q, vector<int>* uspni, vector<int>* lspni, vector<int>* uavri, vector<int>* lavri, vector<int>* umedi, 
 	vector<int>* lmedi, vector<int>* lavr, vector<int>* uavr, vector<int>* lspn, vector<int>* uspn, vector<int>* lmed, vector<int>* umed,
 	vector<int>* num_minmax, vector<int>* num_avr, vector<int>* num_med, vector<int>* tot_spn, vector<int>* tot_avr, int theta, int L) {
-	result.clear();
+    result.clear();
+    result.shrink_to_fit();
```

Comment:
is this shrink_to_fit() new? What does it do, can you say a few words

Reply:
Yes, so I am trying to also clear the vector's capacity. `clear()` removes the elements but the capacity, the allocated memory, would still be there. So here I am doing a little more to clean up the memory.